### DISCUSSION TITLE
I deployed 3.3.3 according to the official documentation, but when I try to boot the installation system on another client, it prompts tftp file not foud

### DISCUSSION THREAD

**Discussion by HUUA** [cobbler/cobbler#3297] [Version specific questions] [answered]
The following is a screenshot of my deployment information
My server is centos8 in the virtual machine, and the client is another virtual machine
![image](https://user-images.githubusercontent.com/27875938/207927851-6ed33363-b452-43e8-a57d-9367136cbeac.png)
![image](https://user-images.githubusercontent.com/27875938/207927931-4197a471-f30a-4037-896d-d1b046337e1f.png)
![image](https://user-images.githubusercontent.com/27875938/207928015-37ce838e-f57e-4307-9ba8-fb5c733a46e2.png)
![image](https://user-images.githubusercontent.com/27875938/207928037-f9e76eca-5b78-4640-a94c-e77360270ad4.png)
![image](https://user-images.githubusercontent.com/27875938/207928347-5356c2d2-a22d-4fe2-a045-2955ceeca0ff.png)

Very eager to get help, thank you very much

**Comment by SchoolGuy** [comment, marked as answer]
So two things:

1. Did you run at any point `cobbler mkloaders`? I don't see the Grub and Syslinux binaries that are required for a successful boot.
2. Could you please check in the TFTP server log what file was requested from the server? Depending on the settings of the TFTP server it could be that the "secure" mode (see [here](https://linux.die.net/man/8/tftpd)) needs to be enabled. The other option is that the DHCP config is not set up fully and thus requires a little bit of tweaking.

**Comment by HUUA** [comment]
yes i found this command before your reply and now it boots fine thanks a lot
